doc/releases/changelog-0.9.0.md

```diff
@@ -64,8 +47,8 @@
   }
   dev = qml.device("lightning.qubit", wires=2)
 
-  @pipeline(my_passes)
-  @qnode(dev)
+  @pipeline(pass_pipeline=my_passes)
```

Comment:
@paul0403 I noticed odd behaviour with the pipeline decorator. It seems that it only accepts named/keyword arguments (likely due to the `*` being used in the signature). This differs from how other decorators work in Catalyst (e.g., qjit).

Would we be able to fix this prior to the release? At the moment, it results in most of the documentation examples for `pipeline` erroring.